HippUnfoldHCP/PostHippUnfoldHCP.sh

```diff
@@ -0,0 +1,300 @@
+    ${CARET7DIR}/wb_command -add-to-spec-file ${AtlasHippUnfoldFolder}/${Subject}.${Mesh}.wb_spec ${HemiStructure} ${AtlasHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_flat.${Mesh}.surf.gii
+    
+    #GIFTI Metrics
+    #Don't add GIFTI to Specs
+    for Scalar in $Scalars ; do
+      Name=`echo $Scalar | cut -d "@" -f 3`
+      Color=`echo $Scalar | cut -d "@" -f 2`
+      Scalar=`echo $Scalar | cut -d "@" -f 1`
+      cp $HippUnfoldFolderOut/hippunfold/sub-${Subject}/surf/sub-${Subject}_hemi-${Hemisphere}_space-${Modality}_den-${Mesh}_label-${Structure}_${Scalar}.shape.gii ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii
+      ${CARET7DIR}/wb_command -set-structure ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii ${HemiStructure}
+      PALETTE ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii ${Color} metric ${CARET7DIR}/wb_command
+      ${CARET7DIR}/wb_command -set-map-names ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii -map 1 "${Subject}_${Name}"
+      if [ $Scalar = "surfarea" ] ; then
+        ${CARET7DIR}/wb_command -surface-vertex-areas ${AtlasHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_midthickness.${Mesh}.surf.gii ${AtlasHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii
+      else
+        cp ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii ${AtlasHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Scalar}.${Mesh}.shape.gii #TODO: mv to have maps in AtlasFolder like Cerebral Cortex?
+      fi
+    done
+    
+    #GIFTI Labels
+    #Don't add GIFTI to Specs
+    for Label in $Labels ; do
+      Name=`echo $Label | cut -d "@" -f 2`
+      Label=`echo $Label | cut -d "@" -f 1`
+      if [ ${Structure} = "hipp" ] ; then
+        if [ ${Hemisphere} = "L" ] ; then
+          Expression="Var*1"
+        elif [ ${Hemisphere} = "R" ] ; then
+          Expression="Var + 8"
+        fi
+        ${CARET7DIR}/wb_command -metric-math "${Expression}" ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Label}.${Mesh}.shape.gii -var Var $HippUnfoldFolderOut/hippunfold/sub-${Subject}/surf/sub-${Subject}_hemi-${Hemisphere}_space-${Modality}_den-${Mesh}_label-${Structure}_${Label}.label.gii
+        ${CARET7DIR}/wb_command -metric-label-import ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Label}.${Mesh}.shape.gii $HCPPIPEDIR/global/config/HippUnfoldLut.txt ${PhysicalHippUnfoldFolder}/${Subject}.${Hemisphere}.${Structure}_${Label}.${Mesh}.label.gii
+      elif [ ${Structure} = "dentate" ] ; then
+        if [ ${Hemisphere} = "L" ] ; then
+          Expression="Var*0+1*6"
+        elif [ ${Hemisphere} = "R" ] ; then
+          Expression="Var*0+1*6 + 8"
```

Comment:
```suggestion
          Expression="6"
        elif [ ${Hemisphere} = "R" ] ; then
          Expression="6 + 8"
```
or `"14"`, if the construction of the value isn't particularly meaningful to understanding.  Just because you need an input file to set the output dimensions doesn't mean the expression has to use it.  There is a special case deliberately to not issue an "unused file" warning when there are no variables and only 1 input file.

Reply:
I didn't know that was possible.  It is written in the fslmaths conventions.